pygments/lexers/_stan_builtins.py

```diff
@@ -436,122 +601,51 @@
     'multi_normal_prec',
     'multi_student_t',
     'multinomial',
+    'multinomial_logit',
     'neg_binomial',
     'neg_binomial_2',
     'neg_binomial_2_log',
+    'neg_binomial_2_log_glm',
     'normal',
+    'normal_id_glm',
     'ordered_logistic',
+    'ordered_logistic_glm',
+    'ordered_probit',
     'pareto',
     'pareto_type_2',
     'poisson',
     'poisson_log',
+    'poisson_log_glm',
     'rayleigh',
     'scaled_inv_chi_square',
+    'skew_double_exponential',
     'skew_normal',
+    'std_normal',
     'student_t',
     'uniform',
     'von_mises',
     'weibull',
     'wiener',
-    'wishart',
+    'wishart'
 )
 
 RESERVED = (
-    'alignas',
```

Comment:
Are these words no longer special at all, or (as I suspect) are they just no longer reserved as identifiers? In the latter case, you should highlight them as `Token.Builtin`.

In any case, please make sure this list of words is covered by the test; I can't find `Keyword.Reserved` in the token output.

Reply:
These genuinely are meaningless now. They were reserved previously because all Stan identifiers needed to be valid C++ identifiers, which has been relaxed. 

I can add them to the test file

Reply:
Ah, OK. What I would like to see in the test file is something using one reserved keyword, so not one of those that are being removed but one that is staying.